stable/redis/README.md

```diff
@@ -313,7 +313,7 @@ $ helm install --name my-release -f ./values-production.yaml stable/redis
 - Number of slaves:
 ```diff
 - cluster.slaveCount: 2
-+ cluster.slaveCount: 3
++ cluster.slaveCount: 4
```

Comment:
you haven't updated this value in values-production.yaml so this change is not correct.